### PR TITLE
Fix not using the worker working directory

### PR DESCRIPTION
**Pull request by dralley**
[noissue]

### CODE REVIEW

**Comment by dralley** [comment]
Q: does it need to be fixed here instead or is this code intending to use `/tmp/`

File: pulp_container/app/tasks/builder.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def build_image_from_containerfile(
     containerfile = Artifact.objects.get(pk=containerfile_pk)
     repository = ContainerRepository.objects.get(pk=repository_pk)
     name = str(uuid4())
-    with tempfile.TemporaryDirectory(".") as working_directory:
```
Comment (dralley) [inline pulp_container/app/tasks/builder.py:116]: First kwarg is not `dir`, it's `suffix`, even though the former would make way more sense : /

**Review by lubosmj — APPROVED** [review]

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
Actually this may need a changelog entry. Do we want to backport it?

**Comment by ipanova** [comment]
@dralley both of instances are tasks so they meant to use workers' dir. Please file an issue. We'd want to backport this to 2.11

**Comment by dralley** [comment]
@ipanova @mdellweg Done

**Comment by dralley** [comment]
My Q: was intended to apply to this line https://github.com/pulp/pulp_container/blob/b1725ad6bb8803dc293cd9938cac5bde443f19e5/pulp_container/app/models.py#L494=

**Comment by ipanova** [comment]
> My Q: was intended to apply to this line https://github.com/pulp/pulp_container/blob/b1725ad6bb8803dc293cd9938cac5bde443f19e5/pulp_container/app/models.py#L494=

no, this one is correct, this code path can be called outside of task.

**Pull request merged**